README.md

```diff
@@ -123,4 +123,31 @@ We re-expose this component from [@solid/react](https://github.com/solid/react-c
   <LogoutButton/>
 ```
 
+### Uploader
+This component allows you to upload files in your POD. This is using React render prop patterns.
+
+The Uploader component contains the file uploading logic and Solid integration, but does not include the UI. You may pass in your own UI as demonstrated here:
+
+```javascript
+<Uploader
+  {...{
+    fileBase: "https://example.org/public",
+    render: (props) => (
+      <ProfileUploader {...{ ...props }} />
+    )
+  }}
+/>
+```
+
+Props  | Type | Default | Description
+------------- | ------------- | ------------- | -------------
+fileBase  | String  |  null |  Files Destination
```

Comment:
Does this mean the URL that the uploader will assign to the uploaded file, or the container that it will upload the file to? (I suspect the latter.)

In general I think we should be as verbose as necessary when it comes to these fields.